### PR TITLE
Fix horizontal gallery button icon directions in rtl mode

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
Ensure in rtl, the chevron right/lefts are flipped to correctly indicate the page direction

# Why
In rtl currently the icons point the opposite direction

# How Tested

| before | after |
| -- | -- |
| image | image |

### CODE REVIEW

File: packages/react-components/src/theming/icons.tsx [hunk 1]
```diff
@@ -206,6 +205,16 @@ const DataLossPreventionProhibited16Regular = (): JSX.Element => {
   return <Prohibited16Regular />;
 };
 
+const HorizontalGalleryLeftButton = (): JSX.Element => {
+  const rtl = useTheme().rtl;
+  return rtl ? <ChevronRight20Regular /> : <ChevronLeft20Regular />;
+};
+
+const HorizontalGalleryRightButton = (): JSX.Element => {
+  const rtl = useTheme().rtl;
```
Comment (prabhjot-msft) [inline packages/react-components/src/theming/icons.tsx:214]: Does this return a boolean?

**Review by prabhjot-msft — APPROVED** [review]
Small question, otherwise looks good!

File: packages/react-components/src/theming/icons.tsx (hunk 1 above)
Comment (JamesBurnside) [reply packages/react-components/src/theming/icons.tsx:214]: Yep! Really it should be called isRtl, or better `dir: 'ltr' | 'rtl`

**Review by mgamis-msft — APPROVED** [review]

**Comment by mgamis-msft** [comment]
Maybe in another PR but could we do the same thing for VerticalGalleryLeftButton and VerticalGalleryLeftButton to fix the rtl snapshots like these:
https://github.com/Azure/communication-ui-library/blob/main/packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/hermetic/OverflowGallery.test.ts-snapshots/overflow-gallery-with-many-participants-rtl-Desktop-Chrome-16-9-linux.png

**Comment by JamesBurnside** [comment]
> Maybe in another PR but could we do the same thing for VerticalGalleryLeftButton and VerticalGalleryLeftButton to fix the rtl snapshots like these: [`main`/packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/hermetic/OverflowGallery.test.ts-snapshots/overflow-gallery-with-many-participants-rtl-Desktop-Chrome-16-9-linux.png](https://github.com/Azure/communication-ui-library/blob/main/packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/hermetic/OverflowGallery.test.ts-snapshots/overflow-gallery-with-many-participants-rtl-Desktop-Chrome-16-9-linux.png?rgh-link-date=2023-05-03T18%3A01%3A12Z)

O great catch, I'll update this PR to do the same thing for those icons

**Comment by JamesBurnside** [comment]
> Maybe in another PR but could we do the same thing for VerticalGalleryLeftButton and VerticalGalleryLeftButton to fix the rtl snapshots like these: [`main`/packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/hermetic/OverflowGallery.test.ts-snapshots/overflow-gallery-with-many-participants-rtl-Desktop-Chrome-16-9-linux.png](https://github.com/Azure/communication-ui-library/blob/main/packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/hermetic/OverflowGallery.test.ts-snapshots/overflow-gallery-with-many-participants-rtl-Desktop-Chrome-16-9-linux.png?rgh-link-date=2023-05-03T18%3A01%3A12Z)



> > Maybe in another PR but could we do the same thing for VerticalGalleryLeftButton and VerticalGalleryLeftButton to fix the rtl snapshots like these: [`main`/packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/hermetic/OverflowGallery.test.ts-snapshots/overflow-gallery-with-many-participants-rtl-Desktop-Chrome-16-9-linux.png](https://github.com/Azure/communication-ui-library/blob/main/packages/react-composites/tests/browser/snapshots/beta/tests/browser/callwithchat/hermetic/OverflowGallery.test.ts-snapshots/overflow-gallery-with-many-participants-rtl-Desktop-Chrome-16-9-linux.png?rgh-link-date=2023-05-03T18%3A03%3A35Z)
> 
> O great catch, I'll update this PR to do the same thing for those icons

Updated 👍

**Pull request merged**